static/shared/js/poll_data.js

```diff
@@ -95,8 +95,19 @@ export class PollData {
 
             inbound: (sender_id, data) => {
                 const idx = data.idx;
-                const key = sender_id + "," + idx;
                 const option = data.option;
+
+                if (!Number.isInteger(idx) || idx < 0 || idx > 99999) {
```

Comment:
This number seems pretty arbitrary. How'd you pick it?

Reply:
artibrarily...the value could be much smaller for any sane pool

Reply:
(now using constant w/smaller upper limit and comment)